src/OpenTelemetry.Extensions/Logs/OpenTelemetryLoggingExtensions.cs

```diff
@@ -34,4 +36,52 @@ public static OpenTelemetryLoggerOptions AttachLogsToActivityEvent(
         return loggerOptions.AddProcessor(new ActivityEventAttachingLogProcessor(options));
 #pragma warning restore CA2000 // Dispose objects before losing scope
     }
+
+    /// <summary>
+    /// Adds the <see cref="BaggageLogRecordProcessor"/> to the <see cref="OpenTelemetryLoggerOptions"/>.
```

Comment:
nit: wondering if should even mention `BaggageLogRecordProcessor` given its an implementation detail and is not public? It might be possible to implement this differently in the future, and we can avoid implementation details in doc comments.